packages/terra-search-field/tests/nightwatch/DelayedSearchField.jsx

```diff
@@ -13,12 +16,14 @@ class DelayedSearchField extends React.Component {
 
   render() {
     return (
-      <div>
-        <div id="search-callback-text">
-          Search Text: {this.state.searchText}
+      <Base locale={locale}>
+        <div>
+          <div id="search-callback-text">
+            Search Text: {this.state.searchText}
+          </div>
+          <SearchField searchDelay={1000} onSearch={(searchText) => { this.setState({ searchText }); }} />
         </div>
```

Comment:
Is this outer div needed?

Reply:
It could be removed now that we wrapped the example in the `<Base>` component.

Reply:
Removed unnecessary div.
https://github.com/cerner/terra-core/pull/1206/commits/bca7c3a5dc1fc9c6e61dee6c5d2b52f13ea9eca8